src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -116,6 +131,45 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
     }
   };
 
+  const renderReasonInput = (index: number) => {
+    const typeValue = watch(`why.${index}.type`);
+    return typeValue === `${ChangeRequestReason.OtherProject}` ? (
+      <Controller
+        key={index}
+        name={`why.${index}.explain`}
+        control={control}
+        render={({ field: { onChange, value } }) => (
+          <Autocomplete
+            id="other-project-autocomplete"
+            isOptionEqualToValue={(option, value) => option.id === value.id}
+            options={projectOptions}
+            size="small"
+            placeholder="Select a project"
+            value={projectOptions.find((element) => element.id === value) || null}
+            sx={{ mx: 1, flexGrow: 1, borderRadius: 2, height: '100%' }}
+            renderInput={(params: AutocompleteRenderInputParams) => (
+              <TextField sx={{ height: '100%' }} {...params} label="Project" />
```

Comment:
<img width="872" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/15f2a971-4863-4a77-b92a-7fe1b16fa465">
the project autcomplete is smaller than the select next to it. Looks weird/ makes the delete icon off center, I'd make it the same size